src/auslib/util/comparison.py

```diff
@@ -14,7 +14,15 @@ def has_operator(value):
     return value.startswith(("<", ">"))
 
 
+def glob_op(value, operand):
+    """Compare a string or StrictVersion against a glob stripped of its `*`.
+    Currently only supported in version_compare."""
+    return value.startswith(operand)
+
+
 def get_op(pattern):
+    if pattern.endswith(".*"):
```

Comment:
For posterity, I was going to suggest that we don't require the preceeding `.` to glob, which would let us do things like `8*` to match 80-89. After some thought I think it's an anti-feature, since it would also match things like `800`, which feels like a footgun.

Reply:
+1. That reminds me, I'll grab the [version100](https://github.com/mozilla-releng/balrog/pull/1924/files#diff-70e892168832c6627bc1b314d2a287db1f4ac842e66c89e6a8f8525a2c765ce7R86) bug fix from the previous PR :)